docs/installation.rst

```diff
@@ -2,62 +2,53 @@
 * `montage_wrapper`: STIPS uses montage to generate mosaics. It is only imported if
   STIPS is asked to generate a multi-detector image.
 * `numpy`: STIPS uses numpy extensively for almost everything that it does.
 * `photutils`: STIPS uses photutils to determine the flux inside the half-light radius
   in generated Sersic profiles.
-* `synphot` and `stsynphot`: STIPS uses synphot and stsynphot to generate 
+* `synphot>=1.1.1` and `stsynphot>=1.1.0`: STIPS uses synphot and stsynphot to generate
   bandpasses, count rates, and zero points. Note that the reference data must
   also be downloaded, as described below in "Doanloading Required Data".
 * `scipy`: STIPS uses scipy to manipulate its internal images (zoom and rotate).
 
-Finally, STIPS requires a set of data files whose location is marked by setting the environment
-variable `stips_data`. The current version of the STIPS data is located on box and can be downloaded via the link below.
-
-Downloading STIPS Data
-#######################
-
-STIPS needs data for reference and calibration. The latest version of the STIPS data can be downloaded as follows::
-
-    # Use wget ot curl to download the data
-    $ wget https://stsci.box.com/shared/static/iufbhsiu0lts16wmdsi12cun25888nrb.gz -O stips_data.tar.gz
-
-    # Unpack the data
-    $ tar -xzvf stips_data.tar.gz
-
-    # Point the stips environment variable in your bash profile or in terminal
-    $ export stips_data=<absolute_path_to_this_folder>/stips_data
-
+Finally, STIPS requires a set of data files whose location is marked by setting the environment variable `stips_data`, which will be installed as 
+part of these instructions.
 
 Installing Using Conda and Source
 ##################################
 
 STIPS can be installed using the source code and a Conda environment file.
-If you do not have anaconda or miniconda installed, please visit the `astroconda docs <https://astroconda.readthedocs.io/en/latest/getting_started.html>`_ for installation instructions.
-We have included a Conda environment file for easily installing or updating Conda packages to meet STIPS requirements.
-Please follow the steps below to install STIPS:
+If you do not have anaconda or miniconda installed, please visit the `anaconda docs <https://docs.anaconda.com/anaconda/install/>`_ for 
+installation instructions.  We have included a Conda environment file for easily installing or updating Conda packages to meet STIPS 
+requirements.  Please follow the steps below to install STIPS:
 
 Installing
 **********
 
-1. You will need to clone the STIPS source code from the `spacetelescope/STScI-STIPS <https://github.com/spacetelescope/STScI-STIPS.git>`_ repository.`cd` into the directory you would like to store the source code and run::
+1. You will need to clone the STIPS source code from the `spacetelescope/STScI-STIPS <https://github.com/spacetelescope/STScI-STIPS.git>`_ repository.  `cd` into the directory you would like to store the source code and run::
```

Comment:
I recommend replacing the "Installing" section with the following:
```
#. You will need to clone the STIPS source code from the `spacetelescope/STScI-STIPS <https://github.com/spacetelescope/STScI-STIPS.git>`_ repository.  `cd` into the directory you would like to store the source code and run::

    git clone https://github.com/spacetelescope/STScI-STIPS.git

    cd STScI-STIPS

#. The environment file can be used in two ways:

   * To create a new Conda environment named ``stips``::

        conda env create -f environment.yml
        conda activate stips


   * Or, to install to or update an existing and currently active Conda environment::

        conda env update --file environment.yml


#. You can now install STIPS using the cloned source code as follows::

    python setup.py install
```
- Using `#.` automates the numbering process for the outer list.
- Swapping the inner list from numbered to bulleted helps demonstrate that the two options are separate and not to be done in sequence.
- More use of two carats on each side of one to get monospaced text.

Reply:
Removed the third bullet (on running `python setup.py install`) because the environment file installs STIPS itself.